pages/osx/uuidgen.md

```diff
@@ -1,6 +1,6 @@
 # uuidgen
 
-> Generate new UUID (Universally Unique IDentifier) strings.
+> Generate new UUID (Universally Unique Identifier) strings.
```

Comment:
This should not be changed. The whole point of the extra capitalisation here is to reference the acronym `UUID` here.

Reply:
No problem.  I didn't think it was a good idea to change.

Reply:
```suggestion
> Generate new UUID (Universally Unique IDentifier) strings.
```